pkg/services/cosmosdb/common-types.go

```diff
@@ -3,7 +3,7 @@ package cosmosdb
 import "github.com/Azure/open-service-broker-azure/pkg/service"
 
 type provisioningParameters struct {
-	IPFilterRules *ipFilterRule `json:"ipFilters"`
+	IPFilterRules *ipFilterRule `json:"ipRangeFilters"`
```

Comment:
I think the docs didn't get this same change.

Reply:
I think I like the original name of `ipFilters` better...

My rationale is that this object also include options `allowAccessFromAzure` and `allowAccessFromPortal`, which (as a user) don't feel like ranges.

Reply:
ok, will revert :)

Reply:
In a similar vein, I wonder if the `filters` property of this object should maybe be renamed to `allowedIPRanges`